src/applications/edu-benefits/0994/prefill-transformer.js

```diff
@@ -12,9 +12,26 @@ export function prefillTransformer(pages, formData, metadata) {
     emailAddress: formData.emailAddress,
   };
 
+  // Determine if bank information page should start in edit mode
+  const {
+    bankAccountType,
+    bankAccountNumber,
+    bankRoutingNumber,
+    bankName,
+  } = formData;
```

Comment:
`bankAccount` comes in the prefill as 
```
bankAccount: {
          bankAccountType: 'Checking',
          bankAccountNumber: '*********1234',
          bankRoutingNumber: '*****2115',
          bankName: 'Comerica'
        },
``` 
(this is an issue as the property names don't match the schema definition of bankAccount) see `vets-api/config/form_profile_mappings/22-0994.yml`

You will need to map `formData.bankAccount`'s properties to `formData['view:bankAccount'].bankAccount` 's properties that are defined in `vets-json-schema/dist/22-0994-schema.json` .  

When empty view fields are removed on submit-transformer should handle pulling `bankAccount` back up a layer but you will need to verify this.